purchases/src/main/kotlin/com/revenuecat/purchases/identity/IdentityManager.kt

```diff
@@ -59,7 +58,7 @@ internal class IdentityManager(
         val cacheEditor = deviceCache.startEditing()
         deviceCache.cacheAppUserID(appUserIDToUse, cacheEditor)
         subscriberAttributesCache.cleanUpSubscriberAttributeCache(appUserIDToUse, cacheEditor)
-        invalidateCustomerInfoAndETagCacheIfNeeded(appUserIDToUse, cacheEditor)
+        invalidateETagCacheIfNeeded(appUserIDToUse)
```

Comment:
Note that I'm still invalidating the ETag cache if the customer info in cache isn't valid, this is because we don't want to reuse an existing response in this case, and we want to receive a full new response